examples/crowdfunding/scripts/transfer/claim.js

```diff
@@ -0,0 +1,52 @@
+const { TransactionType, SignType, toBytes } = require('algob');
+const { executeTransaction } = require('./common');
+
+async function run (runtimeEnv, deployer) {
+  const masterAccount = deployer.accountsByName.get('master-account');
+  const creatorAccount = deployer.accountsByName.get('alice');
+
+  await executeTransaction(deployer, {
+    type: TransactionType.TransferAlgo,
+    sign: SignType.SecretKey,
+    fromAccount: masterAccount,
+    toAccountAddr: creatorAccount.addr,
+    amountMicroAlgos: 5000000,
+    payFlags: {}
+  });
+
+  // App argument to claim.
+  const appArgs = [toBytes('claim')];
+
+  // Get AppInfo and AssetID from checkpoints.
+  const appInfo = deployer.getSSC('crowdFundApproval.teal', 'crowdFundClear.teal');
+
+  // Get Escrow Account Address
+  const escrowAccount = await deployer.loadLogic('crowdFundEscrow.py', [], { APP_ID: appInfo.appID });
+
+  // Atomic Transaction (Stateful Smart Contract call + Payment Transaction)
+  const transactions = [
```

Comment:
```suggestion
  const txGroup = [
```

Reply:
Let's make it clear that these are atomic transactions. Probably we will need to update the `executeTransaction` docs as well.

Reply:
ok